src/Storages/Kafka/StorageKafka.h

```diff
@@ -117,7 +115,7 @@ class StorageKafka final : public shared_ptr_helper<StorageKafka>, public IStora
     std::list<std::shared_ptr<ThreadStatus>> thread_statuses;
 
     /// Handle error mode
-    HandleKafkaErrorMode handle_error_mode;
+    // HandleKafkaErrorMode handle_error_mode; // NOLINT -- actually not used, needs checing
```

Comment:
It's in settings now and it wasn't deleted, you can delete it.

Reply:
Done, tnx.